provider-directory-nginx-proxy/documentation/dev.provider-directory.metadata.json

```diff
@@ -1 +1,176 @@
-{}
+{
+  "id": "health-api-provider-directory-r4-capability",
+  "resourceType": "CapabilityStatement",
+  "version": "0",
+  "name": "API Management Platform | Provider Directory | FHIR R4",
+  "status": "draft",
+  "date": "2021-03-29T10:05:52Z",
+  "publisher": "Department of Veterans Affairs",
+  "contact": [
+    {
+      "name": "API Support",
+      "telecom": [
+        {
+          "system": "email",
+          "value": "api@va.gov"
+        }
+      ]
+    }
+  ],
+  "description": "FHIR R4 Clinical Health APIs.",
+  "kind": "instance",
+  "software": {
+    "name": "health-api-provider-directory-r4"
+  },
+  "fhirVersion": "4.0.0",
+  "format": [
+    "application/json",
+    "application/fhir+json"
+  ],
+  "rest": [
+    {
+      "mode": "server",
+      "security": {
+        "extension": [
+          {
+            "url": "http://fhir-registry.smarthealthit.org/StructureDefinition/oauth-uris",
+            "extension": [
+              {
+                "url": "token",
+                "valueUri": "https://sandbox-api.va.gov/services/provider-directory/v0/r4/token"
+              },
+              {
+                "url": "authorize",
+                "valueUri": "https://sandbox-api.va.gov/oauth2/provider-directory/v1/authorization"
+              },
+              {
+                "url": "manage",
+                "valueUri": "https://sandbox-api.va.gov/oauth2/provider-directory/v1/manage"
+              },
+              {
+                "url": "revoke",
+                "valueUri": "https://sandbox-api.va.gov/oauth2/provider-directory/v1/revoke"
+              }
+            ]
+          }
+        ],
+        "cors": true,
+        "service": [
+          {
+            "coding": [
+              {
+                "system": "https://www.hl7.org/fhir/valueset-restful-security-service.html",
+                "code": "SMART-on-FHIR",
+                "display": "SMART-on-FHIR"
+              }
+            ],
+            "text": "SMART-on-FHIR"
+          }
+        ],
+        "description": "http://docs.smarthealthit.org/"
+      },
+      "resource": [
+        {
+          "type": "Location",
+          "profile": "http://hl7.org/fhir/us/core/StructureDefinition-us-core-location.html",
+          "interaction": [
+            {
+              "code": "search-type",
+              "documentation": "Implemented per specification. See http://hl7.org/fhir/R4/http.html"
+            },
+            {
+              "code": "read",
+              "documentation": "Implemented per specification. See http://hl7.org/fhir/R4/http.html"
+            }
+          ],
+          "versioning": "no-version",
+          "referencePolicy": [
+            "literal",
+            "local"
+          ],
+          "searchParam": [
+            {
+              "name": "_id",
```

Comment:
For each resource, we should list the same search parameters that data-query supports. Take a look at:
 `curl -s https://sandbox-api.va.gov/services/fhir/v0/r4/metadata`